### DISCUSSION TITLE
Error message memory

### DISCUSSION THREAD

**Discussion by Piratenkanon** [AzuraCast/AzuraCast#5095] [Q&A]
i have installed AzuraCast on a Debian 10 server with 16gb RAM and 300gb SSD
Plesk Obsidian 18.0.41  Docker 1.5.2-231
installation was successful
as i wil start the containers i have this error message:

Error: {"message":"OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:385: applying cgroup configuration for process caused: mkdir /sys/fs/cgroup/memory/docker/a6c52ab0ef9a39b4a5fff6f63e224f9a47b9d837426a879507283e4145d0df20: cannot allocate memory: unknown"}

Anyone an idea???

greetings, Martin

**Comment by soundboxfm** [comment]
i don't have the same problem.
I had RAM 14 GB after a short time this is fully exhausted. Have increased to 16 GB, and again after a few hours it is permanently exhausted.
I'm now at 18 GB and still the same problem again.
What can I do?
